### ISSUE TITLE
Questions about training detail

### ISSUE THREAD

**Issue by qiminchen** [xiumingzhang/GenRe-ShapeHD#33]
Hi,

I am wondering when training the marrnet2 with ShapeNet synthetic images, did you train the Marrnet2 only with ground truth normal, depth or did you train the Marrnet2 with ground truth normal, depth for some epochs and then keep training Marrnet2 with predictions from Marrnet1.

And did you use `canon_sup` when training Marrnet2? What are the `batch_size` and `epoch_batches` (how many data used in training per epoch?) For example, when training chair class, did you use 4 x 2500 training points or 6778 x 20 all training points (ignore invalid point)? Because if I use all the training point per epoch, the Eval loss fluctuate a lot and the reconstruction results are not good.

And are these setups the same for GenRe?

Thanks

**Comment by xiumingzhang** [comment]
Hi. You should be able to find all these training details in the invocation scripts in https://github.com/xiumingzhang/GenRe-ShapeHD/tree/master/scripts. If there's anything unanswered, please feel free to reopen this issue.

**Comment by qiminchen** [comment]
Thanks, one more question: did you train the Marrnet2 ONLY with ground truth normal, depth or did you train the Marrnet2 with ground truth normal, depth for **some epochs** and then resume training Marrnet2 with the predictions from Marrnet1 instead?

**Comment by xiumingzhang** [comment]
Yes, we trained our Net 2 with ground-truth maps for some epochs and switched to predicted maps for some finetuning. Adding @ztzhang in case I misremember something.

**Comment by qiminchen** [comment]
If that's the case, after how many epochs do you switch to predicted maps?

**Comment by xiumingzhang** [comment]
`canon_sup` is the flag indicating whether the supervision is in the canonical pose or not. 

If you are talking about MarrNet or ShapeHD, we had this flag on, as these two models predict voxels in the canonical pose. 

if you are talking about GenRe, we had this flag off, since GenRe predicts view-aligned voxels.
